code/src/components/Home.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import NoProjects from "./NoProjects";
+import ProjectList from "./ProjectList";
+import { Container } from "styledelements/elements";
+
+const Home = () => {
+  const projects = useSelector((store) => store.projects.project);
+  return (
+    <Container>
+      {projects.length > 0 ? <ProjectList /> : <NoProjects />}
```

Comment:
This is a really nice feature of conditional rendering in the navbar. Making it possible to show all created lists and toggle between them is making the app more user friendly. Very good work!